exporter/exporterhelper/internal/queuebatch/config.go

```diff
@@ -5,12 +5,27 @@ package queuebatch // import "go.opentelemetry.io/collector/exporter/exporterhel
 
 import (
 	"errors"
+	"runtime"
 	"time"
 
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/exporter/exporterbatcher"
 )
 
+func NewDefaultConfig() Config {
+	return Config{
+		Enabled:         true,
+		Sizer:           exporterbatcher.SizerTypeItems,
+		NumConsumers:    runtime.NumCPU(),
+		QueueSize:       100_000,
+		BlockOnOverflow: true,
+		Batch: &BatchConfig{
+			FlushTimeout: 200 * time.Millisecond,
+			MinSize:      2048,
```

Comment:
Given this default, users who want to disable batching are expected to set this to `0`, right?

Reply:
Clarified, this is a default only for tests. Also helped me find the other bug.